### DISCUSSION TITLE
Decoding the input files

### DISCUSSION THREAD

**Discussion by mmikhasenko** [ComPWA/polarimetry#6] [Show and tell]
I just [implemented](https://github.com/redeboer/polarization-sensitivity/blob/main/julia/notebooks/model0.jl) the decoding in julia.
The idea is to have a numerical cross check, + we compare the speed of the two implementations once it matches.

It only reads isobars. The contraction with the coupling is not yet implemented.

**Comment by mmikhasenko** [comment]
I see now that it should have been a pull request. Sorry, will do more accurate text time

**Reply by redeboer** [reply]
Nah, don't worry. No strict conventions on this repo ^^
